build/rust/Cargo.lock

```diff
@@ -1501,15 +1525,21 @@ dependencies = [
 
 [[package]]
 name = "multihash"
-version = "0.16.2"
+version = "0.16.3"
```

Comment:
Why is multihash being bumped here? Please remove any Cargo.lock changes that are not strictly necessary for this.

Reply:
it looks like rolling this back would eliminate most of the extraneous changes to Cargo.lock here

Reply:
multihash 0.16.3 is required by fvm_shared. The last release compatible with our current 0.16.2 is fvm_shared@v0.9.0, released in August.

@cypt4 Do you know if the ripemd hash is necessary to support for the new address type?

Reply:
it looks like [blake2b](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0048.md#the-f4-address-class) is used

Reply:
looking at Cargo.lock for multihash, the only change it should be triggering is adding ripemd and hex-literal. If we can reduce it down to just the 0.16.2 -> 0.16.3 bump and those two additional deps (nothing else should need to be bumped) then it's fine